spring-xd-shell/src/test/java/org/springframework/xd/shell/command/AbstractFileSourceAndFileSinkTests.java

```diff
@@ -40,7 +40,7 @@
  * @author David Turanski
  * @author Franck Marchand
  */
-public class FileSourceAndFileSinkTests extends AbstractStreamIntegrationTest {
+public class AbstractFileSourceAndFileSinkTests extends AbstractStreamIntegrationTest {
 
 	private static final File DEFAULT_IN = new File("/tmp/xd/input");
```

Comment:
This isn't going to work on windows... I guess it was excluded from the build before?
Maybe `new File(System.getProperty("java.io.tmpdir"));`